boards/nxp/ls1046ardb/ls1046ardb_ls1046a_defconfig

```diff
@@ -1,9 +1,7 @@
+# Copyright 2024 NXP
```

Comment:
in ls1046ardb_ls1046a.yaml remove nxp_ from identifier

Reply:
fixed, thanks.